src/style-auro-ld.scss

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// general WCSS imports
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/utilityMixins/focusVisible";
+@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/animation";
+
+// Import Auro SCSS variables
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariables";
+
+// import button styles into element namespaced wrapper
+auro-button-light {
+  @import './style-auro.scss';
+}
+
+// light DOM support for hover states
+auro-button-light:not(.is-touching) {
+  .auro-button {
+    &:hover {
+      @include auro_transition(all, 150ms, ease-in-out);
+      
+      background-color: var(--auro-color-ui-hover-on-light);
+      border: 1px solid var(--auro-color-ui-hover-on-light);
+    }
+
+    &--secondary {
+      &:hover {
+        @include auro_transition();
+
+        background-color: var(--auro-color-background-lighter);
+        border: 1px solid var(--auro-color-ui-default-on-light);
+      }
+    }
+
+    &--tertiary {
+      &:hover {
+        @include auro_transition();
+
+        background-color: var(--auro-color-background-lighter);
+        border-color: transparent;
+      }
+    }
+
+    &Ondark {
+      &:hover {
+        @include auro_transition(all, 150ms, ease-in-out);
```

Comment:
unnecessary; hover will be inherited from auro-button. When defining modifiers, you only need to define the modification from the base selector.